public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -305,14 +314,20 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
                   <Stack gap={0.5}>Evaluation interval</Stack>
                 </Label>
               }
-              invalid={!!errors.groupInterval}
+              invalid={Boolean(errors.groupInterval) ? true : undefined}
               error={errors.groupInterval?.message}
             >
-              <Input
-                id="groupInterval"
-                placeholder="1m"
-                {...register('groupInterval', evaluateEveryValidationOptions(rulesWithoutRecordingRules))}
-              />
+              <Stack direction="column">
+                <Input
+                  id="groupInterval"
+                  placeholder="1m"
```

Comment:
Should the placeholders also be based on the `minInterval`? Or is that overkill?

Reply:
Good idea! Let's also default to the smallest interval so the user has to type fewer things.